README.md

```diff
@@ -1,5 +1,49 @@
-# integration-adaptor-lab-results
-Integration Adaptor to simplify processing of Pathology and Screening results.
+# Lab Results - Pathology and Screening Adaptor
+
+## Adaptor Scope
+
+The main objective of the Lab Results Adaptor is to hide complex legacy standards and instead present a simple and consistent interface aligned to current NHSD national standards. The adaptor removes the requirement for a GP System to handle the complexities of EDIFACT and MESH messaging.
+
+At a high level, the Lab Results Adaptor exposes a queue from which the GP System can consume medical reports
+
+See the [Resources](#resources) section for links to the underlying services and standards. 
+
+Adaptor handles two types of messages
+
+- Pathology `NHS003`
+- Screening `NHS004`
+
+## Adaptor API
+
+The GP System will receive medical reports on the `Outbound GP Queue` in form of a FHIR DSTU3 Bundle.
+
+### Examples
+
+Examples of both Pathology and Screening messages are provided as part of the adaptor's User Acceptance Tests.
+Each example is built from 3 files:
+
+- `<example-id>.edifact.dat` - message provided by laboratory
+- `<example-id>.fhir.json` - message translated to FHIR format
+- `<example-id>.nhsack.dat` - confirmation messasge sent back to the laboratory
+
+Pathology examples can be found here //TODO
+Screening examples can be found here //TODO
+
+## Adaptor Architecture
+
+//TODO link to digital.nhs.uk
```

Comment:
Can this be done? If not, do we have a task in the backlog for it?

Reply:
https://gpitbjss.atlassian.net/browse/NIAD-993